src/courtroom.cpp

```diff
@@ -894,9 +897,9 @@ void Courtroom::enter_courtroom(int p_cid)
   }
 
   if (ao_app->custom_objection_enabled &&
-      (file_exists(ao_app->get_character_path(current_char, "custom.gif")) ||
-      file_exists(ao_app->get_character_path(current_char, "custom.apng"))) &&
-      file_exists(ao_app->get_character_path(current_char, "custom.wav")))
+      (file_exists(ao_app->get_character_path(current_char, "custom1.gif")) ||
+      file_exists(ao_app->get_character_path(current_char, "custom1.apng"))) &&
+      file_exists(ao_app->get_character_path(current_char, "custom1.wav")))
```

Comment:
This will break characters with only one custom objection.

Reply:
I know, as I said in the description all custom objections need to be renamed in order to avoid having too many if statements if you get what I mean.